BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -5471,21 +5515,27 @@ IEnumerator ReloadRoutine()
 
             UpdateGUIWeaponState();
         }
-        IEnumerator ChargeRoutine()
+        IEnumerator ChargeRoutine(bool decharge = false)
```

Comment:
Did you mean `discharge` or were you intending to use French https://en.wiktionary.org/wiki/d%C3%A9charge ?

Reply:
de-charge/uncharge. Discharge is probably clearer, mais *decharge* dans francais travaille aussi.